src/main/java/io/supertokens/webserver/api/multitenancy/BaseCreateOrUpdate.java

```diff
@@ -938,6 +939,17 @@ private static TenantConfig applyTenantUpdates_5_0(TenantConfig tenantConfig, Js
         return tenantConfig;
     }
 
+    private static void validateFirstFactorsName(TenantConfig tenantConfig) throws ServletException {
```

Comment:
I think u could allow uppercase, underscore as well

Reply:
You need to validate required secondary factors also the same way

Reply:
done. both of them.